Dockerfile.ubuntu

```diff
@@ -19,16 +19,10 @@ FROM $BASE_IMAGE as base_build
 
 LABEL version="1.0.0"
 
-ARG INSTALL_DIR=/opt/intel/openvino
-ARG INSTALL_DIR=/opt/intel/openvino
-ARG TEMP_DIR=/tmp/openvino_installer
-ARG DL_INSTALL_DIR=/opt/intel/openvino/deployment_tools
-ARG DL_DIR=/tmp
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
+
 ARG JOBS
 ARG APT_OV_PACKAGE=openvino-2022.1.0
-ARG CHECK_COVERAGE=0
-ARG NVIDIA=0
-ARG http_proxy
```

Comment:
Why are some args removed while others are kept?

Reply:
Unused args are removed, some of them are moved below. This will make use of cache if somebody changes the argument